GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_parquet.yaml

```diff
@@ -0,0 +1,9 @@
+subclass_name: dc2_dm_catalog.DC2DMCatalog
+base_dir: /global/projecta/projectdirs/lsst/production/DC2_ImSim/Run2.1i/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary
+schema_filename: dpdd_schema.yaml
+filename_pattern: 'dpdd_object_tract_\d+\.parquet$'
+description: DC2 Run 2.1i DPDD Object Table
+creators: ['DESC DC2 Team']
+pixel_scale: 0.2
+include_in_default_catalog_list: true
```

Comment:
Should we include this in the default catalog list? Maybe too confusing?

Reply:
Oh, yes.  Good call.  Someday I mean to seamlessly transition.  But no need to confuse for now.  People who really want the Parquet file access often just want the Parquet files directly (Dask, Spark, e.g.,)